### DISCUSSION TITLE
Hyperdrive or dat for CLI usage?

### DISCUSSION THREAD

**Discussion by vasantm** [beakerbrowser/beaker#1640] [General] [answered]
Hello, We (a bioinfomatics lab) recently started exploring dat and using it for sharing datasets.
The team is made of people comfortable at the CLI and other bench scientists who rather use a browser to explore/download data. 
My idea was to create dat shares for the informatics folks and point other scientists to beaker browser to explore those same data shares.
With the recent release of Beaker Browser and the announcement that you're moving to hyperdrive. What's the best use practice for achieving what we want?
Should we switch to hyperdrive at the commandline as well

**Comment by pfrazee** [comment, marked as answer]
We're still deciding as a community what to do with the `dat` CLI. I somewhat think it'd be wise to keep it on the previous protocol for people to access `dat://` data, but that may not be what the community decides.

In the meantime, I'd advise using the `hyperdrive-daemon` now instead of `dat`. You can find it at https://github.com/hypercore-protocol/hyperdrive-daemon

**Comment by vasantm** [comment]
Thanks Paul,  for the clarification and prompt response. 👍
I think we might test out hypercore to see how it fits our needs
